Source/Utils/Average.cpp

```diff
@@ -27,3 +27,99 @@ Average::ToCellCenter ( MultiFab& mf_out,
                      } );
     }
 }
+
+void
+Average::CoarsenAndInterpolateLoop ( MultiFab& mf_cp,
+                                     const MultiFab& mf_fp,
+                                     const int dcomp,
+                                     const int scomp,
+                                     const int ncomp,
+                                     const IntVect ratio )
```

Comment:
I think it would be nice to use the name `crse_ratio`, as we use it in other places and it is more explicit.
```suggestion
                                     const IntVect& crse_ratio )
```